### ISSUE TITLE
the initial cost is 0

### ISSUE THREAD

**Issue by ghost** [log0/neural-style-painting#5]
the initial cost is 0 so no optimization happens

**Comment by xiaoyu-meng-mxy** [comment]
same problem. Is there any solutions for this?

**Comment by xiaoyu-meng-mxy** [comment]
maybe you should delete line 283

**Comment by ashwin-999** [comment]
i'm running it on notebook, and i'm facing the same problem. Any fix?

**Comment by robbierolin** [comment]
Cast the constants in the loss functions to floats
e.g.
`return (1 / float(4 * N**2 * M**2)) * tf.reduce_sum(tf.pow(G - A, 2))`

**Comment by amseet** [comment]
Hi, 
I did the changes you recommended by @robbierolin, but now I get nan values for sum and black patches on the image.

Can anyone help?

Thank you in advance.
